src/Core/src/Core/ISearchBar.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.Maui
+{
+    public interface ISearchBar : IView, IText
```

Comment:
What you have done is totally correct, however, we would also have the `TextColor` property in SearchBar not implemented. Could you change this to implement only `IView` and add the Text property in ISearchBar?. Again, what you have done is the right way, but until we have colors, etc. implemented we would only show implemented properties in the control.

Reply:
Done!